draft-ietf-tls-tls13.md

```diff
@@ -2348,17 +2354,17 @@ can behave in one of two ways:
   to accept only a subset of the early data messages.
 
 - Request that the client send another ClientHello by responding with a
-  HelloRetryRequest.  A client MUST NOT include the "early_data" extension in
+  HelloRetryRequest. A client MUST NOT include the "early_data" extension in
   its followup ClientHello.
 
-In order to accept early data, the server server MUST have accepted a
-PSK cipher suite and selected the the first key offered in the
-client's "pre_shared_key" extension. In addition, it MUST verify that
-the following values are consistent with those negotiated in the
-connection during which the ticket was established.
+In order to accept early data, the server MUST have accepted a PSK cipher suite
```

Comment:
There's no such thing as a "PSK cipher suite" anymore. Please update this text while you're here. I guess just drop this part and make it "...server MUST select the first key offered..."